src/excalidraw-app/index.tsx

```diff
@@ -121,6 +121,7 @@ const initializeScene = async (opts: {
     }
   }
   if (roomLinkData) {
+    trackEvent("share", "session join");
```

Comment:
is that the right place for join?

Reply:
What do we wanna track? If joins in general, then https://github.com/excalidraw/excalidraw/blob/00ac804dd867d214c455292397206233fc784157/src/excalidraw-app/collab/Portal.tsx#L26. If only room creations, then https://github.com/excalidraw/excalidraw/blob/master/src/excalidraw-app/collab/CollabWrapper.tsx#L190